### DISCUSSION TITLE
Focusing on vac bypass

### DISCUSSION THREAD

**Discussion by syndrexhub** [danielkrupinski/Osiris#2433] [Ideas]
I think you dont have to add anything like other player want. Because this cheat is already perfect and good, better focus on vac bypass and trusted mode for this cheat.

**Comment by MoldyCSGO** [comment]
Vac bypass loader by him will make it so you can't get vacced and bypassing trusted mode requires an injector that patched NtOpenFile() or uses manual mapping, such as csghost on unknown cheats.

**Reply by osirisbestlegitcheat** [reply]
> Vac bypass loader by him will make it so you can't get vacced and bypassing trusted mode requires an injector that patched NtOpenFile() or uses manual mapping, such as csghost on unknown cheats.

THATS THE EXACT COMBO I USE LOL

**Reply by Overheadcams** [reply]
so I just inject the Osiris DLL with csghost. that's enough to get past VAC?

**Reply by linuxer9** [reply]
> 
> 
> so I just inject the Osiris DLL with csghost. that's enough to get past VAC?

AFAIK, csghost is detected. you shouldn't use it without vac bypass

**Reply by Overheadcams** [reply]
what should we use besides csghost?

**Reply by linuxer9** [reply]
> 
> 
> what should we use besides csghost?

this https://github.com/danielkrupinski/VAC-Bypass-Loader

**Reply by Overheadcams** [reply]
> 
> 
> > so I just inject the Osiris DLL with csghost. that's enough to get past VAC?
> 
> AFAIK, csghost is detected. you shouldn't use it without vac bypass

every time I run the vac bypass loader, it creates an exe on my desktop called dontshare or something, and malwarebytes detects it as a when I scan it manually.

**Reply by linuxer9** [reply]
> 
> every time I run the vac bypass loader, it creates an exe on my desktop called dontshare or something, and malwarebytes detects it as a when I scan it manually.

It shouldn't create any thing on your desktop. I don't know what binaries are you using. regardless, just download the source from the link I provided and compile it yourself.
It's normal for an antivirus or an antimalware to detect it as a suspicious file (due to it's nature ofc) just add an exception
